titannet/chain.json

```diff
@@ -0,0 +1,126 @@
+    },
+    "genesis": {
+      "genesis_url": "https://github.com/Titannet-dao/titan-chain/releases/download/v0.3.0/genesis.json"
+    },
+    "sdk": {
+      "type": "cosmos",
+      "version": "v0.50.10",
+      "tag": "v0.50.10"
+    },
+    "ibc": {
+      "type": "go",
+      "version": "v8.5.1"
+    },
+    "cosmwasm": {
+      "version": "v0.53.0",
+      "repo": "https://github.com/CosmWasm/wasmd",
+      "tag": "v0.53.0"
+    }
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/titanchain/images/ttnt.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/titanchain/images/ttnt.svg"
+  },
+  "description": "Titan Chain aims to solve the resource utilization challenges in the Web3 ecosystem by creating an efficient, transparent, and decentralized infrastructure marketplace. Built on Cosmos SDK, it enables seamless trading of computing, storage, and bandwidth resources while ensuring fair compensation for resource providers.\n\nThe project focuses on making decentralized infrastructure services more accessible and efficient through its modular blockchain design. Using the Inter-blockchain Communication (IBC) protocol, Titan Chain can connect with other blockchain networks, facilitating broader resource sharing and integration across the ecosystem.\n\nThe network uses TTNT tokens for governance and securing the network through a Delegated Proof-of-Stake consensus mechanism, where token holders can participate in network decisions and earn rewards for contributing to network stability.",
+  "peers": {
+    "seeds": [
+      {
+        "id": "bb075c8cc4b7032d506008b68d4192298a09aeea",
+        "address": "47.76.107.159:26656",
+        "provider": "titan"
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "b656a30fd7585c68c72167805784bcd3bed2d67c",
+        "address": "8.217.10.76:26656"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://rpc.titannet.io:443",
+        "provider": "Titan"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://lcd.titannet.io:443",
+        "provider": "Titan"
+      }
+    ],
+    "grpc": [
+      {
+        "address": "https://grpc.titannet.io:443",
+        "provider": "Titan"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "guru",
+      "url": "https://testnet.titan.explorers.guru",
+      "tx_page": "https://testnet.titan.explorers.guru/transaction/${txHash}",
+      "account_page": "https://testnet.titan.explorers.guru/account/${accountAddress}",
+      "validator_page": "https://testnet.titan.explorers.guru/validator/${validatorAddress}",
+      "proposal_page": "https://testnet.titan.explorers.guru/proposal/${proposalId}",
+      "block_page": "https://testnet.titan.explorers.guru/block/${blockHeight}"
+    }
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/titanchain/images/ttnt.png",
```

Comment:
if the images are relocated to /testnets/titannettestnet/images/, then all these path must also be updated. For now, it is permissible to leave the images under /titannet/images/, but many teams add them images to the testnet images directory first, and then move them over to the mainnet one once the mainnet is registered.